src/hats/io/parquet_metadata.py

```diff
@@ -41,7 +41,7 @@ def write_parquet_metadata(
             defaults to `catalog_path` if unspecified
         create_thumbnail (bool): if True, create a data thumbnail parquet file for
             the dataset. Defaults to True.
-        pixel_threshold (int): the number of points per partition specified when
+        thumbnail_threshold (int): the number of points per partition specified when
             importing the catalog. Defaults to 1_000_000.
```

Comment:
```suggestion
        thumbnail_threshold (int): maximum number of rows in the data thumbnail,
            which is otherwise the number of partitions. Defaults to 1_000_000.
```

In this new context, and with this name, this description may be better.

Reply:
Thank you, Derek! :)